tests/unit/test_nautobot_inventory.py

```diff
@@ -243,3 +243,38 @@ def test_nornir_nautobot_device_groups(device, expected_groups):
         )
 
     assert test_nornir.inventory.hosts[device].groups == expected_groups
+
+
+@pytest.mark.parametrize("device", ["den-dist01", "den-wan01", "den-dist02"])
```

Comment:
Do you mind splitting this into two separate tests? I think it'll help make the test more explicit as to what it is testing as well if the default option works as intended.

Looking at it again, maybe adding additional asserts within this test would be fine as well to make sure each specific `InitNornir` is being tested for the desired result as well.

I'll leave the implementation up to you.

Reply:
Yeah sure can do that. So you mean provide: 
-  one test for keys of devices with `pynautobot_dict=True`
-  one test for keys of devices with `pynautobot_dict=False`

Reply:
Correct. Just to make sure each one is returning the dictionary or not.

It could be just adding those asserts to this test as well.